tests/testthat/test-ruleset_construction.R

```diff
@@ -0,0 +1,69 @@
+test_that("Rules can be added together", {
+  rule_1 <- rule(mpg >10)
+  rule_2 <- rule(hp >10)
+  expect_equal(
+               rule_1 + rule_2,

```

Comment:
Can you reindent the lines? (If you are using RStudio: CTRL + A then CTRL + I)

Reply:
Sure thing! Assuming we want what's inside the tests indented another time.